src/rhsmlib/cloud/providers/azure.py

```diff
@@ -39,26 +40,36 @@ class AzureCloudProvider(BaseCloudProvider):
     # for very long time. It would be good to update the version from time to time,
     # because old versions (three years) are deprecated. It would be good to update
     # the API version with every minor version of RHEL
-    API_VERSION = "2020-09-01"
+    API_VERSION = "2021-02-01"
 
-    CLOUD_PROVIDER_METADATA_URL = "http://169.254.169.254/metadata/instance?api-version=" + API_VERSION
+    BASE_CLOUD_PROVIDER_METADATA_URL = "http://169.254.169.254/metadata/instance?api-version="
```

Comment:
Could this ip address change in the future? Should we put it some singular place and reference it instead?

Reply:
It is interesting that all three main cloud providers use this link local IPv4 address for IMDS servers. So, this address is (de facto) standard. Thus I don't think that they will change it. Maybe they will add support for IPv6 in the future. We can put it to some singular place in the future during some refactoring.